docker/azure-functions/release.sh

```diff
@@ -23,7 +23,10 @@ do
         --build-arg PYTHON_VERSION=$version \
         -t bentoml/azure-functions:$BENTOML_VERSION-py${version//.} \
         .
-    docker push bentoml/azure-functions:$BENTOML_VERSION-py${version//.}
+
+    if ! [[ -t 1 ]]; then
```

Comment:
allow developers to build Docker images. We can assume that this scripts will run in an interactive session when dev invokes it rather than `./dev/release-docker-images.sh`